lib/modules/userTagger.js

```diff
@@ -225,6 +165,54 @@ addModule('userTagger', function(module, moduleID) {
 		}
 	};
 
+	function addTagger() {
+		var ignoreToggleButton = RESUtils.createElement.toggleButton(null, 'userTaggerIgnore', false, 'yes', 'no');
+		var ignoreSettingsLink = modules['settingsNavigation'].makeUrlHashLink('userTagger', 'hardIgnore', ' configure ', 'gearIcon');
+		var $colorSelect;
+
+		// Base HTML.
+		RESTemplates.load('userTaggerDialog', function(template) {
+			var output = template.html();
+			$('body').append(output);
+		});
+
+		// Define $tagger.
+		$tagger = $('#userTaggerToolTip');
+		// Add generated elements.
+		$tagger.find('.res-usertag-ignore').append(ignoreToggleButton).append(ignoreSettingsLink);
+		// Populate color selection with options.
+		$colorSelect = $tagger.find('#userTaggerColor');
+		for (var color in bgToTextColorMap) {
+			var bgColor = (color === 'none') ? 'transparent' : color;
+			$colorSelect.append('<option style="background-color: ' + bgColor + '; color: ' + bgToTextColorMap[color] + ' !important;" value="' + color + '">' + color + '</option>');
```

Comment:
You should be able to use [a list](https://mustache.github.io/mustache.5.html#Sections) in the template to accomplish this, i.e. something like (haven't actually tested it):
```html
<select id="userTaggerColor">
  {{#colors}}
    <option style="background-color: {{bgColor}}; color: {{textColor}} !important;" value="{{color}}">{{color}}</option>
  {{/colors}}
</select>
```
and then when you call `template.html()`, pass in an object of this shape:
```js
template.html({
  colors: [
    { bgColor, textColor, color }
  ]
})
```